### PR TITLE
[doc] Fix typos in cgi.rst

### PR DESCRIPTION
**Pull request by geryogam**
Fix two typos in the documentation of the `cgi` standard library.

### CODE REVIEW

File: Doc/library/cgi.rst [hunk 1]
```diff
@@ -347,7 +347,7 @@ Caring about security
 .. index:: pair: CGI; security
 
 There's one important rule: if you invoke an external program (via the
-:func:`os.system` or :func:`os.popen` functions. or others with similar
+:func:`os.system` or :func:`os.popen` functions or others with similar
```
Comment (iritkatriel) [inline Doc/library/cgi.rst]:
```suggestion
:func:`os.system`, :func:`os.popen` other functions with similar
```

File: Doc/library/cgi.rst [hunk 2]
```diff
@@ -424,7 +424,7 @@ above on installing your CGI script carefully can save you a lot of time.  If
 you wonder whether you have understood the installation procedure correctly, try
 installing a copy of this module file (:file:`cgi.py`) as a CGI script.  When
 invoked as a script, the file will dump its environment and the contents of the
-form in HTML form. Give it the right mode etc, and send it a request.  If it's
+form in HTML form. Give it the right mode etc., and send it a request.  If it's
```
Comment (iritkatriel) [inline Doc/library/cgi.rst]: form in HTML format?

Comment (geryogam) [reply Doc/library/cgi.rst]: I agree. Fixed.

File: Doc/library/cgi.rst (hunk 1 above)
Comment (geryogam) [reply Doc/library/cgi.rst]: Thanks, I agree. Updated.

**Review by iritkatriel — APPROVED** [review]

**Comment by maggyero** [comment]
Thank you for the review @iritkatriel.

**Pull request merged**